test/multi-window-test.js

```diff
@@ -24,15 +24,15 @@ describe('multiple windows', function () {
   it('launches the application', function () {
     return app.client
       .getWindowCount().should.eventually.equal(2)
-      .windowByIndex(1)
+      .windowByIndex(0)
```

Comment:
Not sure why the order has changed, but don't think this change breaks the idea of the test.

Reply:
Actually, it looks like we can't rely on the order of `windowhandles`

https://github.com/webdriverio/webdriverio/blob/567f8778bc8a66a2a68ad9e12da2c81e86fabe47/packages/webdriver/protocol/webdriver.json#L192

So I rewrote the test to be independent of the order of the returned windows.